hyprbars/main.cpp

```diff
@@ -59,27 +59,38 @@ Hyprlang::CParseResult onNewButton(const char* K, const char* V) {
 
     Hyprlang::CParseResult result;
 
-    // hyprbars-button = color, size, icon, action
+    // hyprbars-button = bgcolor, size, icon, action, fgcolor
 
     if (vars[0].empty() || vars[1].empty()) {
-        result.setError("var 1 and 2 cannot be empty");
+        result.setError("bgcolor and size cannot be empty");
         return result;
     }
 
     float size = 10;
     try {
         size = std::stof(vars[1]);
     } catch (std::exception& e) {
-        result.setError("failed parsing var 2 as int");
+        result.setError("failed to parse size");
         return result;
     }
 
-    auto X = configStringToInt(vars[0]);
-    if (!X) {
-        result.setError("var2 is not a valid number");
+    auto bgcolor = configStringToInt(vars[1]);
+    auto fgcolor = configStringToInt("");
```

Comment:
this is invalid so it will crash later at *fgcolor

Reply:
works on my machine, but will change anyway

Reply:
![image](https://github.com/user-attachments/assets/6eadebed-955c-43dd-a320-65f706f90088)
Maybe you aren't on git. Anyways, just throw a `"0"` there.

Reply:
fixed with commit 57b7940dead80a2c5c2513281653fed8311d35f5